packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/Device/AddAccountSingleChainDialog.tsx

```diff
@@ -0,0 +1,247 @@
+import {
+  LangDisplay,
+  ScrollableContainer,
+  FlexGapContainer,
+  DialogBox,
+  DialogBoxHeader,
+  DialogBoxBody,
+  LeanBoxContainer,
+  LeanBox,
+  Typography,
+  Image,
+  InputLabel,
+  DialogBoxFooter,
+  Button,
+  settingsIcon,
+  Flex,
+  bitcoinIcon,
+} from '@cypherock/cysync-ui';
+import { Toggle } from '@cypherock/cysync-ui/dist/esm/components/atoms/Toggle';
```

Comment:
Do not import from subpath, only import from root.
```suggestion
import { Toggle } from '@cypherock/cysync-ui';
```

Reply:
yes changed